httpclient5-cache/src/main/java/org/apache/hc/client5/http/impl/cache/CacheConfig.java

```diff
@@ -257,11 +232,11 @@ public int getMaxUpdateRetries(){
     }
 
     /**
-     * Returns whether 303 caching is enabled.
-     * @return {@code true} if it is enabled.
+     * @deprecated No longer applicable. Do not use.
      */
+    @Deprecated
     public boolean is303CachingEnabled() {
-        return allow303Caching;
+        return true;
```

Comment:
@ok2c  Should the return value of the deprecated is303CachingEnabled() method be set to false to align with all the changes?

Reply:
@arturobernalg The cache protocol layer will now be treating 303 responses as potentially cacheable by default. This is why I thought the deprecated method should also return `true`. Does that make sense?

Reply:
@ok2c  that its correct. my bad.